pyudev/_util.py

```diff
@@ -141,3 +142,34 @@ def get_device_type(filename):
         return 'block'
     else:
         raise ValueError('not a device file: {0!r}'.format(filename))
+
+
+def eintr_retry_call(func, *args, **kwargs):
+    """
+    Handle interruptions to an interruptible system call.
+
+    Run an interruptible system call in a loop and retry if it raises EINTR.
+    The signal calls that my raise EINTR prior to Python 3.5 are listed in
+    PEP 0475.  Any calls to these functions must be wrapped in eintr_retry_call
+    in order to handle EINTR returns in older versions of Python.
+
+    This function is based on _eintr_retry_call in python's subprocess.py.
```

Comment:
This should work just as well under Python3.5 as under previous versions, since try block should always succeed immediately. Could you put a note to that effect somewhere in the docstring?